12.nadeo-importer/03.how-to-make-a-specularmap-diffusemap-for-objects/docs.md

```diff
@@ -15,8 +16,27 @@ I will explain very quickly what are the specifications and how to save them in
 
 ## What you need
 
-* a graphic manipulation software (ps, gimp etc.)
+* a graphic manipulation software (Adobe Photoshop (PS), Gimp etc.)
+Free & Open Source: https://www.gimp.org/
```

Comment:
You completely break the bulleted list formatting here - if you want to add information to the same list item, check the syntax for it [here](https://daringfireball.net/projects/markdown/syntax#list).